toolbox/fdc3-compliance/src/appDefinition.json

```diff
@@ -0,0 +1,31 @@
+{
+    "appId": "fdc3-compliance",
+    "name": "FDC3 Compliance",
+    "manifest": "{\"url\":\"http://localhost:8080\"}",
+    "manifestType": "web",
+    "version": "1.2",
+    "title": "FDC3 Compliance",
+    "tooltip": "FDC3 Compliance Test",
+    "description": "FDC3 Compliance Test",
+    "images": [
+      {
+        "url": "https://fdc3.finos.org/img/fdc3-logo-2019.png"
+      }
+    ],
+    "publisher": "FINOS",
+    "icons": [
+      {
+        "icon": "https://fdc3.finos.org/img/fdc3-logo-2019.png"
```

Comment:
This logo doesn't work well as an icon, i'd suggest swapping out for http://fdc3.finos.org/toolbox/fdc3-workbench/fdc3-icon-256.png
(which you can copy into this project)